resolver/auth_mutations.resolvers_test.go

```diff
@@ -95,6 +140,34 @@ func TestLogin(
 		t.Run(
 			tt.name,
 			func(t *testing.T) {
+				if tt.name == ErrorFromConfig {
+					patch := gomonkey.ApplyFunc(config.Load, func() (*config.Configuration, error) {
+						return nil, fmt.Errorf("error in loading config")
+					})
+					defer patch.Reset()
+				}
+
+				var tg jwt.Service
+				if tt.name == ErrorFromJwt {
+					patch := gomonkey.ApplyFunc(service.JWT, func(cfg *config.Configuration) (jwt.Service, error) {
+
+						return tg, fmt.Errorf("error in creating auth service")
+
+					})
+					defer patch.Reset()
+				}
+
+				patch := gomonkey.ApplyFunc(tg.GenerateToken, func(u *models.User) (string, error) {
+					if tt.name == ErrorFromGenerateToken {
+						return "", fmt.Errorf("error in generating token")
+					} else {
+						return "", nil
+					}
+
+				})
+				defer patch.Reset()
```

Comment:
add error check for the response where we check whether the desired error is received or not